packages/screens/LaunchpadERC20/LaunchpadERC20Tokens/LaunchpadERC20CreateToken.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { View } from "react-native";
+
+import { CreateTokenBasics } from "./LaunchpadERC20CreateTokenBasics";
+import { CreateTokenDetails } from "./LaunchpadERC20CreateTokenDetails";
+import { CreateTokenSign } from "./LaunchpadERC20CreateTokenSign";
+import { LaunchpadERC20CreateTokenHeader } from "../component/LaunchpadERC20CreateTokenHeader";
+import { useCreateTokenState } from "../hooks/useCreateToken";
+
+import { Breadcrumb } from "@/components/Breadcrumb";
+import { ScreenContainer } from "@/components/ScreenContainer";
+import { SpacerColumn } from "@/components/spacer";
+import { NetworkKind } from "@/networks";
+import { ScreenFC } from "@/utils/navigation";
+
+const renderStep = (stepIndice: number) => {
+  if (stepIndice === 1) return <CreateTokenBasics />;
+  if (stepIndice === 2) return <CreateTokenDetails />;
+  if (stepIndice === 3) return <CreateTokenSign />;
+};
+
+const STEPS = ["Basics Informations", "Details Informations", "Sign & Deploy"];
+
+export const LaunchpadERC20CreateTokenScreen: ScreenFC<
+  "LaunchpadERC20CreateToken"
+> = () => {
+  const {
+    stepIndice,
+    actions: { gotoStep },
+  } = useCreateTokenState();
```

Comment:
We should not externalise variables if it's not necessary (if they are not used somewhere else). It's better to priorize an encapsulation logic and better scoping.  In addition to increase the lisibility, it increase perfs because these variables are loaded only if the component is loaded (I'm not 100% sure).

**If you are ok, could you make this change in every new TS you've created ?**

```suggestion
export const LaunchpadERC20CreateTokenScreen: ScreenFC<
  "LaunchpadERC20CreateToken"
> = () => {
  const STEPS = ["Basics Informations", "Details Informations", "Sign & Deploy"];

  const {
    stepIndice,
    actions: { gotoStep },
  } = useCreateTokenState();
  
  const renderStep = (stepIndice: number) => {
    if (stepIndice === 1) return <CreateTokenBasics />;
    if (stepIndice === 2) return <CreateTokenDetails />;
    if (stepIndice === 3) return <CreateTokenSign />;
  };
```

Reply:
it's better to put the constant array and simple functions in the global scope actually, otherwise an array and a function are reconstructed on every render. Also they can't be used as effect or memo dependency when declared in the component body

Reply:
So i let it like this and put the comment as resolve